src/pages/home/sidebar/OptionRow.js

```diff
@@ -63,6 +64,12 @@ const propTypes = {
     /** Whether to disable the interactivity of this row */
     disableRowInteractivity: PropTypes.bool,
 
+    /** Addtional styles to be passed to displayNameStyle */
+    textStyles: stylePropTypes,
+
+    /** Addtional styles to be passed to alternateTextStyle  */
+    alternateTextStyles: stylePropTypes,
+
```

Comment:
I don't think that gray color will ever change for OptionRow anywhere so instead of these props use a single prop. Something like `showDisableState` and then add the proper styles inside OptionRow.

Reply:
+1. Please do not mark comments resolved until there is an indication for the same.